src/systems/cyberpunk-red-core.js

```diff
@@ -15,11 +15,15 @@ export default {
   "ITEM_FILTERS": [
     {
       "path": "type",
-      "filters": "role,skill,criticalInjury"
+      "filters": "role, skill, criticalInjury"
     },
     {
       "path": "system.type",
-      "filters": "cyberwareInternal,cyberwareExternal,fashionware"
+      "filters": "cyberwareInternal, cyberwareExternal, fashionware"
+    },

```

Comment:
Justed tested it on a new world. this filter option is not needed anymore with the additional below. It currently prevents placing a few cyberware into piles as well.

Reply:
I'll remove that before release, thanks! It's hard to know what exactly is needed for each system :D